hazelcast/src/test/java/com/hazelcast/internal/dynamicconfig/DynamicConfigRollingUpgradeTest.java

```diff
@@ -1,85 +0,0 @@
-/*
```

Comment:
I needed to delete these tests since they were starting instances with 3.8 and 3.9 versions, trying to dynamically add configuration which was added in later versions. The tests failed because they were looking for `hazelcast-config-3.8.xsd` and `hazelcast-config-3.9.xsd`. I've removed the tests since we no longer need to test if some config is accepted or not in older versions.